source/community/puppet_projects_workflow.markdown

```diff
@@ -182,7 +182,61 @@ Please either (a) update it with the information I've requested and re-assign it
 
 #### Code Insufficient
 
-I've put this ticket's status into "Failed Review" and assigned it to you. Please either (a) update the ticket or pull request with a version of code which addresses the issues, or (b) assign it back to me if you are blocked on the problem and I'll move things forward.
+I’ve put this ticket’s status into “Failed Review” and assigned it to you. Please either (a) update the ticket or pull request with a version of code which addresses the issues, or (b) assign it back to me if you are blocked on the problem and I’ll move things forward.
+
+#### Issue already fixed
+
+Thank you for reporting this. This issue has already been addressed in a later release of Puppet. If you experience this issue against the current version of Puppet, please add a comment to this ticket with your reproduction scenario.
+
+For more info on getting the current version of Puppet Agent, see https://docs.puppet.com/puppet/latest/install_pre.html.
+
+#### Issue probably already fixed
+
+Thank you for reporting this. We believe this issue has already been addressed in a later release of Puppet. If you experience this issue against the current version of Puppet, please add a comment to this ticket with your reproduction scenario.
+
+For more info on getting the current version of Puppet Agent, see https://docs.puppet.com/puppet/latest/install_pre.html.
+
+#### Only affected operating system version is EOL
+
+Thank you for reporting this issue. However, we believe this issue only affects an operating system/version that is no longer supported by Puppet, and will not be addressing this in a future release of Puppet.
+
+For a list of current supported operating systems, please see https://docs.puppet.com/pe/latest/sys_req_os.html.
+
+#### Only affected software version is EOL
+
+Thank you for reporting this issue. However, we believe this issue only affects a version of Puppet that has reached its end of life or is no longer supported.
+
+For more info on getting the current version of Puppet Agent, see https://docs.puppet.com/puppet/latest/install_pre.html.
+
+#### Feature is deprecated
+
+Thank you for reporting this issue. However, we believe this issue only affects a feature in Puppet that is deprecated, and is no longer receiving further improvements.
+
+For documentation on the intended replacement workflow, please see <docs> (if applicable)
```

Comment:
Formatting note: to keep that `<docs>` from being eaten as an HTML tag, wrap in backticks.